pkg/apis/core/types_seed.go

```diff
@@ -297,6 +300,13 @@ type SeedSettingDependencyWatchdogProbe struct {
 	Enabled bool
 }
 
+// SeedSettingTopologyAwareRouting controls certain settings for topology-aware traffic routing in the seed.
+// See https://github.com/gardener/gardener/blob/master/docs/usage/topology_aware_routing.md.
+type SeedSettingTopologyAwareRouting struct {
+	// Enabled controls whether certain Services deployed in the seed cluster should be topology-aware.
```

Comment:
Perhaps you could improve "certain" the doc string with information for which services this applies.